### DISCUSSION TITLE
Seeking Help with Hyperland Dots Configuration

### DISCUSSION THREAD

**Discussion by Dead-Slash** [prasanthrangan/hyprdots#617] [General]
Certainly! Here's a revised version of your GitHub discussion post, incorporating your experience as a newcomer to Linux:

Subject: New to Linux and Grateful for Hyperland Dots – Seeking Configuration Assistance

Hello [Developer's Username],

I hope this message finds you well. As someone new to the Linux world, I've been on a journey of exploration and adaptation, and your Hyperland Dots configuration has been an invaluable part of that experience. I want to express my sincere appreciation for sharing this fantastic dotfile; it's been a game-changer for me.

Being a newcomer, I'm still navigating the intricacies of Linux and recently attempted to adapt the Hyperland Dots to my preferences. While updating the userprefs file, I encountered a minor hiccup that I believe stems from my limited experience. I noticed that configurations from hyperland.conf transitioned seamlessly, which is fantastic for a beginner like me!

However, I encountered challenges with preserving configurations related to the font size in Kitty, the styling in Waybar's style.css, and the format of the clock module in config.jsonc. These aspects, I understand, are often user-specific, but I'm eager to learn how to ensure their smooth integration into my setup.

Since I'm still learning the ropes of Linux and someone mentioned to me that Linux is superior to Windows (which I'm keen to explore further), I would greatly appreciate any guidance or insights you could provide on this configuration matter. Your assistance would not only help me enhance my setup but also contribute to my ongoing learning journey.

Once again, thank you for your incredible work on Hyperland Dots. It has already made a positive impact on my Linux experience, and I'm excited to continue adapting and learning.

Best regards,

**Comment by kRHYME7** [comment]
Hi, 1 thing you can do is to Fork this git repo to your system. In your system you can Try to edit those files. Then if you like to get the latest commits  just git pull then try to  merge and fix any conflicts. It is quite easy from time to time.

**Reply by Dead-Slash** [reply]
Thank you so much for your helpful suggestion! I'll definitely give it a try and fork the git repo to my system. Editing the files locally and then pulling the latest commits, followed by merging and resolving any conflicts, sounds like a great approach. I appreciate your guidance—it's reassuring to know that it can be a manageable process.

**Reply by kRHYME7** [reply]
Also recently I hosted a CLI for Hyprdots where you can manage Hyprdots using that. I will be adding a user control file to set user variables again after a restore, but it would depend on user request. 

Here is the link 
https://github.com/kRHYME7/Hyprdots-ctl

Also It is on AUR, to install ``` paru -Sy hyprdots ```
